R/connect.R

```diff
@@ -62,11 +63,11 @@ setMethod("dbConnect", "MySQLDriver",
   function(drv, dbname = NULL, username = NULL, password = NULL, host = NULL,
     unix.socket = NULL, port = 0, client.flag = 0,
     groups = "rs-dbi", default.file = NULL, ssl.key = NULL, ssl.cert = NULL,
-    ssl.ca = NULL, ssl.capath = NULL, ssl.cipher = NULL, ...) {
+    ssl.ca = NULL, ssl.capath = NULL, ssl.cipher = NULL, timeout=3600, ...) {
```

Comment:
The idea was to set a default value so high that queries are not being time-out in general. This is the behaviour I would expect if a timeout is not set.

In anyone wants to time-out the queries with a lower value, it can be set manually.